src/StaticFAQ/StaticFAQError.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Typography } from '@kiwicom/orbit-components';
+
+import { ContactPageLink } from '../common';
+
+const style = css`
+  .staticFAQError {
+    height: 78vh;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+  h1.title {
+    font-family: Roboto;
+    font-size: 28px;
+    font-weight: bold;
+    line-height: 1.2;
+    color: #171b1e;
+    margin-bottom: 8.4px;
+  }
+  div.contactLink {
+    margin-top: 16.8px;
+  }
+`;
+
+const StaticFAQError = () => (
+  <div className="staticFAQError">
+    <h1 className="title">Something&apos;s not quite right.</h1>
+    <Typography type="secondary" size="large">
+      Please, try refreshing the page. Sign out. Then sign in again. If the
```

Comment:
I would remove this `Sign out. Then sign in again.` 

I know it's an issue for us, but it should be happening only without frontend (there is session verification on frontend)
I also created #157  to solve this.